SimPEG/directives/directives.py

```diff
@@ -184,6 +204,8 @@ class BetaEstimate_ByEig(InversionDirective):
 
     beta0 = None  #: The initial Beta (regularization parameter)
     beta0_ratio = 1e2  #: estimateBeta0 is used with this ratio
+    ninit = 4  #: number of vector for estimation.
```

Comment:
This would change the default behaviour - so this is one to flag and chat about. @thast: I see you posted a notebook with your issue in #935. Did you look at the 3D EM problem? I think using multiple iterations might be quite feasible for cheaper simulations, but might be something to trade-off, specifically for the EM problem

Reply:
Indeed. in #937 , I mention the need to revisit the examples, make sure it works as before.
My reasoning for this default value: Put the default to a reasonable estimator, and leave the user makes the conscious choice of downgrading it if needed.